src/util/util.js

```diff
@@ -16,3 +16,8 @@ export const noop = () => {};
 export const noargs = (f) => () => f();
 
 export const sumUnderThreshold = (list, threshold) => list.reduce((acc, i) => acc + Math.min(i, threshold), 0);
+
+export const getCsrf = () => decodeURIComponent(document.cookie.split(/;[ \t]*/)
+  .map(cookie => cookie.trim())
+  .find(cookie => cookie.startsWith('__csrf'))
```

Comment:
How about adding `=` just to make absolute sure that we're matching the `__csrf` cookie, not another cookie with the same prefix, like `__csrf_but_not_really`?

```suggestion
  .find(cookie => cookie.startsWith('__csrf='))
```